products/chef-workstation.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Chef Workstation
+category: app
+tags: ruby-runtime
+iconSlug: chef
+permalink: /chef-workstation
+versionCommand: chef --version
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: "https://docs.chef.io/release_notes_workstation/#__LATEST__"
+eoasColumn: true
+
+identifiers:
+-   repology: chef-workstation
+
+auto:
+  methods:
+  -   chef-workstation: https://docs.chef.io/release_notes_workstation/
```

Comment:
```suggestion
```

This would require a corresponding script in https://github.com/endoflife-date/release-data/tree/main/src. So either we prepare a PR that will be merged before this one, or we do it after and drop this configuration for now.

Reply:
Removed for now